src/dao/pages/DAOSettings.tsx

```diff
@@ -129,6 +129,12 @@ export function DAOSettings() {
             <Text.P_Small_Medium>Description</Text.P_Small_Medium>
             <Text.P_Small_Regular color={Color.Neutral._700}>{description}</Text.P_Small_Regular>
           </Flex>
+          <Flex direction="column" gap="1">
+            <Text.P_Small_Medium>Info Url</Text.P_Small_Medium>
+            <Link as={ReachLink} textStyle="p small regular link" color={Color.Primary._500} to={infoUrl} isExternal>
```

Comment:
I think it should be `INFO URL` following the same pattern on header for DAO ID and Governance Token

Reply:
Done.